### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by StoverS**

### CODE REVIEW

**Comment by R3fractor** [comment]
Looks good.

File: blastoff.c [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+int main(void) {
+        const int start = 10;
+        for (int i = 0; i<=start; i++) {
```
Comment (defreez) [inline blastoff.c:4]: This should count down from 10 to 0, not go from 0 to 10

File: blastoff.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package main
+
+import "fmt"
+
+func main() {
+	const start = 10
+	for i := 0; i<=start; i++ {
```
Comment (defreez) [inline blastoff.go:7]: Should be 10 to 0, not 0 to 10

File: blastoff.c [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+                printf("%d", i);
+                printf("\n"); }
```
Comment (defreez) [inline blastoff.c:6]: Closing curly brace should be on the next line. Use `clang-format`

**Review by defreez — COMMENTED** [review]
Looks good, other than reversing the direction of the loops. Use `clang-format -i` and `go fmt` to deal with style issues.

**Pull request merged**